motionplan/motionPlanner.go

```diff
@@ -71,23 +62,35 @@ func PlanFrameMotion(ctx context.Context,
 ) ([][]frame.Input, error) {
 	// ephemerally create a framesystem containing just the frame for the solve
 	fs := frame.NewEmptyFrameSystem("")
-	err := fs.AddFrame(f, fs.World())
+	if err := fs.AddFrame(f, fs.World()); err != nil {
+		return nil, err
+	}
+	destination := frame.NewPoseInFrame(frame.World, dst)
+	seedMap := map[string][]frame.Input{f.Name(): seed}
+	solutionMap, err := motionPlanInternal(ctx, logger, destination, f, seedMap, fs, nil, constraintSpec, planningOpts)
 	if err != nil {
 		return nil, err
 	}
+	return FrameStepsFromRobotPath(f.Name(), solutionMap)
+}
+
+func PlanMapMotion(
```

Comment:
Should this include a SLAM map? This looks identical to PlanFrameMotion above

Reply:
I called out in the PR description that I'm not planning against the map yet so unless we determine that is something that needs to be done as a part of this PR theres not a use for it.

We could include it in the function signature though, is that a change that you would like to see here?

Reply:
If we're not planning against a map yet and this isn't used, then it shouldn't exist. We don't want to commit a new public function knowing that we'll be changing the type signature soon.

This should be removed from the PR until it's useful.

Reply:
I'm good with that.  Are you ok merging these changes without planning against the map yet?  I spoke with @tessavitabile about this today and was planning on hiding `GoToInputs` functionality behind an build variable for now (there is precedent for this apparently) so that people don't stumble on this thinking it will work but we can commit it to the codebase

Reply:
As far as I can tell, the body of this PR will in fact work, in the sense of planning for and driving a rover that localizes on a SLAM map? In that case I'm 100% ok with merging without actually planning against the map boundaries; the feature will become richer over time.

It's just the actual `PlanMapMotion` that shouldn't exist until it's in its final-ish form. Easier to add a new public function and change what MoveOnMap calls later, rather than create it now and change public function signatures later.

Reply:
removed `PlanMapMotion` and remoted RSDK-2314 from this PR's description